dali/base/dacsds.cpp

```diff
@@ -1105,13 +1105,13 @@ IPropertyTree *CClientRemoteTree::collateData()
     {
         ChangeTree(IPropertyTree *donor=NULL) { ptree = LINK(donor); }
         ~ChangeTree() { ::Release(ptree); }
-        inline void createTree() { assertex(!ptree); ptree = createPTree(RESERVED_CHANGE_NODE); }
+        inline void createTree() { assertex(!ptree); ptree = createPTree(RESERVED_CHANGE_NODE, ipt_lowmem); }
```

Comment:
am not sure these are good candidte for lowmem/atom ptrees..

These are being created during a commit, serialized then destroyed, so _should_ be smallish and transient.